tests/engines/netcdf4_engine/test_netcdf4_converter_engine.py

```diff
@@ -257,15 +272,37 @@ def test_virtual_from_netcdf_group_1(simple2_netcdf_file, tmpdir):
 def test_virtual_from_netcdf_group_2(simple2_netcdf_file, tmpdir):
     uri = str(tmpdir.mkdir("output").join("virtual3"))
     with netCDF4.Dataset(simple2_netcdf_file.filepath, mode="r") as dataset:
-        from_netcdf_group(dataset, uri, use_virtual_groups=True)
+        from_netcdf_group(dataset, uri, use_virtual_groups=True, coords_to_dims=False)
     assert isinstance(tiledb.ArraySchema.load(f"{uri}_array0"), tiledb.ArraySchema)
     with tiledb.open(uri) as array:
         assert array.meta["name"] == "simple2"
 
 
+@pytest.mark.parametrize("collect_attrs", [(True,), (False,)])
+def test_convert_coord(simple_coord_netcdf_example, tmpdir, collect_attrs):
+    uri = str(tmpdir.mkdir("output").join("sparse_example"))
+    converter = NetCDF4ConverterEngine.from_file(
+        simple_coord_netcdf_example.filepath,
+        coords_to_dims=True,
+        collect_attrs=collect_attrs,
+    )
+    converter.convert_to_group(uri)
+    with tiledb.cf.Group(uri, attr="y") as group:
+        schema = group.array.schema
+        assert schema.sparse
+        data = group.array[:]
+    index = np.argsort(data["x"])
+    x = data["x"][index]
+    y = data["y"][index]
+    assert np.array_equal(x, np.array([-1.0, 2.0, 4.0, 5.0]))
+    assert np.array_equal(y, np.array([1.0, 4.0, 16.0, 25.0]))
```

Comment:
These assertions make sense when you compare it to the `simple_coord_netcdf_example` contents, which is defined in another file. Since this is the only test that uses this fixture, inlining it would make the test a bit easier to understand.

Reply:
I completely agree, but I will save this for a big refactor of the NetCDF4 tests I'm planning on doing soon.